apps/balancer-tools/src/app/apr/pool/(components)/PoolTokensTable.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+import { ExternalLinkIcon } from "@radix-ui/react-icons";
+import Image from "next/image";
+
+import Table from "#/components/Table";
+
+import { PoolStatsData } from "../../api/route";
+
+export default function PoolTokensTable({
+  poolStats,
+}: {
+  poolStats: PoolStatsData;
+}) {
+  return (
+    <>
+      <Table color="blue" shade={"darkWithBorder"}>
+        <Table.HeaderRow>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          <Table.HeaderCell classNames="w-1/6 text-end p-4">
+            Volume
```

Comment:
```suggestion
            Balance
```

Reply:
I'm using balancer's nomenclature there

Reply:
can you provide an example? Maybe you are thinking about the "Volume" on the charts, but on the pool composition they use this nomenclature:

<img width="809" alt="image" src="https://github.com/bleu-studio/balancer-tools/assets/63298341/b4794c4f-e1f3-4895-b7d3-bc40635412a7">